network/wsNetwork.go

```diff
@@ -302,6 +302,8 @@ type msgBroadcaster struct {
 	broadcastQueueBulk     chan broadcastRequest
 	// slowWritingPeerMonitorInterval defines the interval between two consecutive tests for slow peer writing
 	slowWritingPeerMonitorInterval time.Duration
+	// enableVoteCompression controls whether vote compression is enabled
+	enableVoteCompression bool
```

Comment:
usage of this `enableVoteCompression` vs `wn.config. EnableVoteCompression` is not consistent, maybe `wn.enableVoteCompression` should be used everywhere